server/game/cards/06-WoE/Conscription.js

```diff
@@ -0,0 +1,38 @@
+const Card = require('../../Card.js');
+
+class Conscription extends Card {
```

Comment:
dupe?

Reply:
Weird, I don't know how that happened, sorry.  All fixed now, please take a look.